.github/workflows/ci.yml

```diff
@@ -102,6 +105,9 @@ jobs:
       with:
         ruby-version: ${{ matrix.ruby }}
 
+    - run: sudo apt-get -yqq install libxml2-dev libxslt-dev
+      if: startsWith(matrix.ruby, 'truffleruby')
+
     - uses: actions/cache@v2
       with:
         path: gemfiles/vendor/bundle
```

Comment:
I noticed the fairly verbose logic here to cache `bundle install`, did you setup-ruby's `bundler-cache: true`?
Was there any issue with it?
https://github.com/ruby/setup-ruby#caching-bundle-install-automatically has some docs about that and where to set BUNDLE_GEMFILE.